packages/maskbook/src/extension/background-script/ImageShuffleService.ts

```diff
@@ -0,0 +1,109 @@
+// import seedrandom from 'seedrandom'
+import { decodeArrayBuffer, encodeArrayBuffer } from '../../utils/type-transform/String-ArrayBuffer'
+import { downloadUrl } from '../../utils/utils'
+import { buf2Img, img2Buf } from '@dimensiondev/stego-js/cjs/canvas/dom'
+
+const seedrandom = require('seedrandom')
```

Comment:
why not use import?

Reply:
import for some reason doesn’t work, not sure why but apparently it was common amongst others

Reply:
for seedrandom